insalan/cms/tests.py

```diff
@@ -3,37 +3,48 @@
 from django.core.exceptions import ValidationError
 from django.utils.translation import gettext_lazy as _
 
-from insalan.cms.models import (
-    Constant,
-    Content
-)
+from insalan.cms.models import Constant, Content
 
 
 class ContentTestCase(TestCase):
     def test_content_with_constants(self):
-        """ test that we can create a content using constant """
+        """test that we can create a content using constant"""
         Constant.objects.create(name="a", value="je suis la valeur a")
         Constant.objects.create(name="b", value="je suis la valeur b")
-        Content.objects.create(name="content_a", content="j'appelle la constante ${a} et la constante ${b}")
+        Content.objects.create(
+            name="content_a", content="j'appelle la constante ${a} et la constante ${b}"
+        )
 
     def test_content_with_unknow_constants(self):
+        """ test that a ValidationError is raised in case of unknown constant used in a content """
         Constant.objects.create(name="a", value="je suis la valeur a")
-        Content.objects.create(name="content", content="je suis ${a} mais aussi une variable ${random}")
+        Content.objects.create(
+            name="content", content="je suis ${a} mais aussi une variable ${random}"
+        )
         self.assertRaises(ValidationError)
 
     def test_display_undefined_constant_list(self):
-        Content.objects.create(name="content", content="je test ${inconnue} mais aussi une variable ${random}")
-        self.assertRaisesMessage(ValidationError, _("Des constantes non définies sont utilisées: inconnue, random"))
+        """ Test that a ValidationError raised by the usage of unknown constants give the correct list of undefined constants
```

Comment:
you can add a line feed after the `"""` and put the docstring on its own line

Reply:
I have to confess: I let black auto-fixed the code :roll_eyes: